src/materialized/src/bin/materialized/main.rs

```diff
@@ -164,6 +164,12 @@ pub struct Args {
     /// The service orchestrator implementation to use, if any.
     #[structopt(long, hide = true, arg_enum)]
     orchestrator: Option<Orchestrator>,
+    /// Labels to apply to all services created by the orchestrator in the form
+    /// `KEY=VALUE`.
+    ///
+    /// Only valid when `--orchestrator` is specified.
+    #[structopt(long, hide = true)]
+    orchestrator_label: Vec<OrchestratorLabel>,
```

Comment:
Do you have an example of how to specify multiple KV pairs? Do we just pass the arg multiple times?

Nitpick, this also has the possibility of duplicate keys, which isn't legal in labels or HashMaps.

Reply:
Yeah exactly: https://github.com/MaterializeInc/cloud/pull/2636

> Nitpick, this also has the possibility of duplicate keys, which isn't legal in labels or HashMaps.

True, but this is the conventional way to pass this stuff through CLI args. Passing JSON as the arg value is harder to type if you're typing these invocations manually.